### DISCUSSION TITLE
question

### DISCUSSION THREAD

**Discussion by APT-ZERO** [hiddify/Hiddify-Manager#304] [Q&A سوالات رایج] [answered]

**Comment by hiddify-com** [comment, marked as answer]
مشکلی ایجاد نمیشه 
سوالات را در قسمت سوالات بپرسید بهتره
لطفا عنوان مناسبی انتخاب کنید تا افراد دیگری هم بتونند سریع تر این سوال را پیدا کنند

**Comment by arman2012** [comment]
بله میتونه اسم تکراری باشه 
از هر واژه ای هم میشه انتخاب کرد

**Comment by sepehrgh8090** [comment]
سلام دوستان و عزیزان
در تمام متنهای آموزشی اشاره مشخص به راه اندازیی پنل و دستورات مربوطه شده اما من هیج جا ندیدم برای اینکه این پنل را از روی سرور پاک کنیم چه دستوری باید بزنیم
ممنون میشم راهنمایی کنید
